src/server-workers.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2021 RethinkDNS and its authors.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import "./helpers/workers/config.js";
+import * as system from "./system.js";
+import { handleRequest } from "./index.js";
+
+((main) => {
+  system.sub("go", systemUp);
+})();
```

Comment:
CF Workers can't wait for it.... Event fetch event listener has to be registered, event if system is not ready.

```
❯ wrangler dev
Error: Something went wrong with the request to Cloudflare...
No event handlers were registered. This script does nothing.
 [API code: 10021]
```

Reply:
Brrr. Fixing with a "hack".